lib/language_process.dart

```diff
@@ -415,7 +416,29 @@ class LanguageProcessPageState extends State<LanguageProcessPage> {
             const Text('Output:', style: TextStyle(fontSize: 18)),
             const SizedBox(width: 10.0),
             ElevatedButton(
-              onPressed: saveToFile,
+              onPressed: () async {
+                if (_outputController.text.isEmpty) {
+                  ScaffoldMessenger.of(context).showSnackBar(
+                    const SnackBar(content: Text('No output to save.')),
```

Comment:
Perhaps a better way to do this from a user experience perspective is to make the button inactive until there is data to save?

Reply:
Fixed! 
The save buttons are disabled when there is no data to be saved, like below:
<img width="608" alt="disabled save button" src="https://github.com/Ting-TT/MLHubFlutter/assets/111097257/d87718df-a8d9-470c-8478-b86d57fd62c0">
<img width="608" alt="disabled save button" src="https://github.com/Ting-TT/MLHubFlutter/assets/111097257/d993e9fc-9843-46e5-bbb5-7c5e5ff0830a">

When there is data to save, they become clickable:
<img width="608" alt="enabled save button" src="https://github.com/Ting-TT/MLHubFlutter/assets/111097257/cef1dd39-6874-48a0-91a7-407ce69858dc">
<img width="608" alt="enabled save button" src="https://github.com/Ting-TT/MLHubFlutter/assets/111097257/a7c59654-6eb0-45aa-aea9-b1fd18f5b108">